CONTRIBUTING.md

```diff
@@ -185,7 +185,7 @@ Automatically formatted with `gn format`.
 [pr_49801]: https://github.com/flutter/engine/pull/49801
 [code_of_conduct]: https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md
 [contrib_guide]: https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md
-[engine_dev_setup]: https://github.com/flutter/flutter/blob/master/docs/engine/contributing/Setting-up-the-Engine-development-environment.md
+[engine_dev_setup]: https://github.com/flutter/engine/blob/main/docs/contributing/Setting-up-the-Engine-development-environment.md
```

Comment:
Nice! But, since this is now in the same repo, can this just be relative?

Reply:
You're right, didn't know that it's a thing. Fixed, thank you!